README.md

```diff
@@ -379,26 +380,26 @@ Provides an instance of a class which has an onBeforeSend method that can be use
 
 * Is there an example app?
 
-  Yup - clone this repository then run the **app** module of the project.
+  Yup - clone this repository, then run the **app** module of the project.
 
-* Not seeing errors in the dashboard?
+* I'm not seeing errors in the dashboard.
```

Comment:
```suggestion
* I'm not seeing errors in Raygun Crash Reporting.
```

Reply:
Because referring to the Crash Reporting product as a dashboard is not something we do anymore. We have an actual product called Dashboards so this will be confusing to keep!